lib/getdef.c

```diff
@@ -282,7 +282,7 @@ unsigned int getdef_unum (const char *item, unsigned int dflt)
 
 	if (   (getlong (d->value, &val) == 0)
 	    || (val < 0)
-	    || (val > INT_MAX)) {
+	    || (val > UINT_MAX)) {
```

Comment:
If the max is raised to UINT_MAX, then maybe we should call getulong()?

BTW, that would require <https://github.com/shadow-maint/shadow/pull/875>.

Reply:
True, that would be an issue on 32 bit systems. Dropped the commit.